packages/cozy-client/src/store/queries.spec.js

```diff
@@ -89,5 +89,25 @@ describe('queries reducer', () => {
       )
       expect(state).toMatchSnapshot()
     })
+
+    it('should only update queries with the right id selector', () => {
+      const query = new Q({
+        doctype: 'io.cozy.todos',
+        id: TODO_3._id
+      })
+      applyAction(initQuery('b', query))
+      applyAction(
+        receiveQueryResult('a', {
+          data: [TODO_2]
+        })
+      )
+      expect(state).toMatchSnapshot()
+      applyAction(
+        receiveQueryResult('a', {
+          data: [TODO_3]
+        })
+      )
+      expect(state).toMatchSnapshot()
```

Comment:
The inconvenient with `toMatchSnapshot` is that it is hard to understand what is tested. A comment like `b query should be updated now` would help I think.